pkg/section/prefix.go

```diff
@@ -12,12 +12,22 @@ type Custom struct {
 	Prefix string
 }
 
+// CustomSeparator allows you to group multiple custom prefix together in the same section
+// gci diff -s standard -s default -s prefix(github.com/company;gitlab.com/company;companysuffix)
+//
+// Note: it was not possible to use comma as a separator because gci support to provide multiple sections at once
+// gci diff -s standard,default
+const CustomSeparator = ";"
```

Comment:
We can use comma since it should be a part of `prefix`.

Reply:
The fix was very simple.

While the usage of `StringSliceP` was legitimate with `-local` legacy parameter

Because parameters were provided as a comma separated list

We can use `StringArrayP` for `--section`